main.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'request'
+require_relative 'gist_content'
+
+loop do
+  puts "Ingrese nombre del archivo "
+  namefile = gets.chomp
+  puts "Descripcion: "
+  description = gets.chomp
+
+  loop do
+    puts "Quieres el gist publico? Si/No"
+    state = gets.chomp.capitalize
+
+    if state == 'Si'
+      @state = true
+      break
+    elsif state == 'No'
+      @state = false
+      break
+    else
+      puts "Respuesta incorrecta"
+    end
+  end
+
+  values = Received_values.new(namefile, description, @state)
+  values.file_existence
+  request =  Assemble.new(namefile, description, @state, values.content)
```

Comment:
What happens with the request if the filename doesn't exist? Does it get made either way?